PayItForward.Web/AutoMapper/IMapTo.cs

```diff
@@ -0,0 +1,12 @@
+﻿namespace PayItForward.Web.AutoMapper
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Linq;
+    using System.Threading.Tasks;
+
+    public interface IMapTo<T>
```

Comment:
You dont need this for now, as it is not used. We can add this later to another project so it can used from anywhere